qgreenland/config/layers.yml

```diff
@@ -1318,7 +1328,7 @@
     extract_files:
       - 'Icemask_Topo_Iceclasses_lon_lat_average_1km_GrIS.nc'
 
-- <<: *racmo_1km
+- <<: *racmo_1km_masks
```

Comment:
Topo does come from the "Icemask" file, but it's not a mask itself. I don't have a suggestion for a different name for this anchor/reference, though. Will sleep on it